source/Date.cpp

```diff
@@ -143,7 +143,82 @@ string Date::LongString() const
 	return result;
 }
 
-
+string Date::CurrentYear() const
+{
+	int year = Year();
+	return std::to_string(year);
+}
+string Date::CurrentShortDate() const
+{
+	int day = Day();
+	int month = Month();
+	int year = Year();
+	string result = Weekday(day, month, year);
+	result += " ";
+	result += std::to_string(day);
+	result += " ";
+	static const string MONTH[] = {
+			"Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"};
+	result += MONTH[month - 1];
+	return result;
```

Comment:
```suggestion
	static const string MONTH[] = {
			"Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"};
	return Weekday(day, month, Year()) + " " + to_string(day) + " " + MONTH[month - 1];
```
In this case, condensed code is still easy to read + Year() is called once, so you don't need a variable for it.